### PR TITLE
Bump k8s lib to 0.22.7

### PR DESCRIPTION
**Pull request by s1061123**

### CODE REVIEW

File: pkg/utils/cniconfig_test.go [hunk 1]
```diff
@@ -20,17 +20,11 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"os"
 	"path/filepath"
-	"testing"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 )
 
-func TestCNIConfig(t *testing.T) {
```
Comment (phoracek) [inline pkg/utils/cniconfig_test.go:29]: Is this change related?

**Review by phoracek — CHANGES_REQUESTED** [review]
Please use this PR only fro the bump. The test changes and .conf files seem to be unrelated. If they are not, please elaborate in the commit message.

File: pkg/utils/net-attach-def_test.go [hunk 2]
```diff
@@ -40,11 +39,6 @@ func EnsureCIDR(cidr string) *net.IPNet {
 	return net
 }
 
-func TestNetworkAttachmentDefinition(t *testing.T) {
```
[the remark this answers is not in the document]
Comment (s1061123) [reply pkg/utils/net-attach-def_test.go:43]: See above.

File: pkg/utils/cniconfig_test.go (hunk 1 above)
Comment (s1061123) [reply pkg/utils/cniconfig_test.go:29]: Both changes are required to pass CI because after the go.mod changes the following error is happen:

```
$ go test -v ./pkg/utils/
=== RUN   TestCNIConfig
Running Suite: CNI Config
=========================
Random Seed: 1645455922
Will run 10 of 10 specs

~~~~~~~~~~
Ran 10 of 10 Specs in 0.001 seconds
SUCCESS! -- 10 Passed | 0 Failed | 0 Pending | 0 Skipped
--- PASS: TestCNIConfig (0.00s)
=== RUN   TestNetworkAttachmentDefinition
Running Suite: Network Attachment Definition utils
==================================================
Random Seed: 1645455922
Will run 20 of 20 specs

~ Failure [0.000 seconds]
Netwok Attachment Definition manipulations
/home/nrt/tohayash/src/npwg/network-attachment-definition-client-s1061123/pkg/utils/net-attach-def_test.go:48
  test convertDNS [It]
  /home/nrt/tohayash/src/npwg/network-attachment-definition-client-s1061123/pkg/utils/net-attach-def_test.go:50

  You may only call BeforeEach from within a Describe, Context or When

  /home/nrt/tohayash/src/npwg/network-attachment-definition-client-s1061123/pkg/utils/cniconfig_test.go:37
------------------------------
~~~~~~~~~~~~~~~~~~~

Summarizing 1 Failure:

[Fail] Netwok Attachment Definition manipulations [It] test convertDNS
/home/nrt/tohayash/src/npwg/network-attachment-definition-client-s1061123/pkg/utils/cniconfig_test.go:37

Ran 20 of 20 Specs in 0.001 seconds
FAIL! -- 19 Passed | 1 Failed | 0 Pending | 0 Skipped
--- FAIL: TestNetworkAttachmentDefinition (0.00s)
FAIL
FAIL    github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/utils  0.018s
FAIL
```

**Comment by s1061123** [comment]
Reverted pkg/utils changes. CI is not passed due to the revert.

**Review by phoracek — APPROVED** [review]
